luarules/gadgets/scavengers/Blueprints/BYAR/blueprint_tiers.lua

```diff
@@ -1,20 +1,12 @@
 local UDN = UnitDefNames
 local wallChance = 0
-local ScavengerTeamID = Spring.GetGaiaTeamID()
-local teams = Spring.GetTeamList()
-for i = 1, #teams do
-	local luaAI = Spring.GetTeamLuaAI(teams[i])
-	if luaAI and luaAI ~= "" and string.sub(luaAI, 1, 12) == 'ScavengersAI' then
-		ScavengerTeamID = i - 1
-		break
-	end
-end
+local gaiaTeamID = Spring.GetGaiaTeamID()
 
 function BPWallOrPopup(faction, tier)
-	if ScavengerTeamID then
-		wallChance = Spring.GetTeamUnitCount(ScavengerTeamID)
+	if gaiaTeamID then
```

Comment:
Isn't this always true? Gaia team is present, no matter what.

Reply:
the scavengerteamid in this one is inaccurate, ive just renamed all instances of it to gaiateamid